deno.jsonc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Added this to just make it easier to run the ci checks locally via `deno task test`